Library/Formula/subversion.rb

```diff
@@ -1,14 +1,14 @@
 class Subversion < Formula
   homepage "https://subversion.apache.org/"
-  url "http://www.apache.org/dyn/closer.cgi?path=subversion/subversion-1.8.10.tar.bz2"
-  mirror "http://archive.apache.org/dist/subversion/subversion-1.8.10.tar.bz2"
-  sha1 "d6896d94bb53c1b4c6e9c5bb1a5c466477b19b2b"
-  revision 2
+  url "http://www.apache.org/dyn/closer.cgi?path=subversion/subversion-1.8.11.tar.bz2"
+  mirror "http://archive.apache.org/dist/subversion/subversion-1.8.11.tar.bz2"
+  sha1 "161edaee328f4fdcfd2a7c10ecd3fbcd51c61275"
+  revision 0
 
   bottle do
-    sha1 "fea8a67fd177ba418c9185423d4ce8055d8f2ef5" => :yosemite
-    sha1 "ab3d546951f17ce2f2159a4e6bbf0c10b0c6a4c0" => :mavericks
-    sha1 "18a00000ae92b6b6bad79f3b84f0ffcba4e655f7" => :mountain_lion
+#    sha1 "fea8a67fd177ba418c9185423d4ce8055d8f2ef5" => :yosemite
+#    sha1 "ab3d546951f17ce2f2159a4e6bbf0c10b0c6a4c0" => :mavericks
+#    sha1 "18a00000ae92b6b6bad79f3b84f0ffcba4e655f7" => :mountain_lion
```

Comment:
You should leave this as it is. Comment this out would make the bot think it's the real comment.